__tests__/index.test.ts

```diff
@@ -386,6 +388,27 @@ describe('Auth0', () => {
 
         expect(transactionManager.get).toHaveBeenCalledWith(queryState);
       });
+      it('throws error with AuthenticationError', async () => {
+        const { auth0, utils } = await localSetup();
+        const queryResult = { error: 'unauthorized' };
+        utils.parseQueryResult.mockReturnValue(queryResult);
+
+        await expect(auth0.handleRedirectCallback()).rejects.toBeInstanceOf(
+          AuthenticationError
+        );
+      });
+      it('throws error with message from error_description', async () => {
```

Comment:
there's no test to validate we're adding the `error` property to the AuthenticationError object